frontend/src/pages/ActivityReport/Pages/Review/index.js

```diff
@@ -87,7 +87,11 @@ const ReviewSubmit = ({
             error={error}
             onSaveForm={onSaveForm}
           >
-            <Accordion bordered={false} items={items} />
+            <>
+              <h2 className="sr-only">accessability</h2>
+              <h3 className="sr-only">accessability</h3>
```

Comment:
This looks odd to have and h2 and an h3 with the same text next to each other.

Reply:
Yes its very odd :-) . It has to do with overcoming an accessibility scan issue. Headings <h1 h2 h3. h4> have to appear in order.

The new Trussworks Accordion component now writes <h4> tags instead of <h2> so we will need to add h 2-3 above. Right now these tags do NOT show in the app.

Doc here: https://dequeuniversity.com/rules/axe/4.3/heading-order

Reply:
The text within would need to have some meaning since it would be read off.

Also, I know its a pain but I think it would just be better to not add those and change the tagname within the accordions using a useEffect

Reply:
or just make our own accordion :)

Reply:
I have updated the text text inside the headings. 

Also moved them out side of their current location so they only appear once.

I think creating our own Accordion would defeat the purpose of using a USWDS component library.

Reply:
Ok new Accordion component added with tests.